src/firebase/firebase.service.ts

```
@@ -0,0 +1,98 @@
+        const serviceAccount = firebaseConfig();
+        if (
+          !serviceAccount ||
+          typeof serviceAccount !== 'object' ||
+          !serviceAccount.private_key
+        ) {
+          throw new Error('Invalid Firebase service account configuration.');
+        }
+        admin.initializeApp({
+          credential: admin.credential.cert(serviceAccount),
+        });
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)